css/collection-card.styl

```diff
@@ -36,13 +29,16 @@
       font-style: normal
 
     .description
-      text-overflow: ellipsis
       color: COPY
       font-size: 0.9em
       height: 150px
       margin: 0.6em 0
       overflow: hidden
+      text-overflow: ellipsis
       text-align: left
+      display: -webkit-box;
```

Comment:
I added the styles below to get the ellipsis to display, following the example [here](https://css-tricks.com/line-clampin/#article-header-id-0). Not sure if there's a better way, but just having `overflow: hidden` and `text-overflow: ellipsis` was not rendering the ellipsis for text that was longer than the area allotted for the description.

Also, as I was working on this, it occurred to me to ask: we're currently limiting description text to 300 characters - is there anywhere besides on the collection card where the description is displayed? Approximately 125 characters fit on the card before the text overflows. Would it make sense to limit the text to, say, 125? Or if I'm just missing where the full 300 character description is displayed, never mind.

Reply:
Well, no, not yet, since the description is new. The description probably should also be displayed on the collection view page too. Maybe somewhere below the name but above the subjects? You could check in with @beckyrother